### PR TITLE
First connectedMachine module generated with the GA swagger

### PR DESCRIPTION
**Pull request by dolauli**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ConnectedMachine/Az.ConnectedMachine.nuspec [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Az.ConnectedMachine</id>
```
Comment (wyunchi-ms) [inline src/ConnectedMachine/Az.ConnectedMachine.nuspec:4]: Is `.nuspec` necessary for the module? The previous ones don't need this.

**Review by wyunchi-ms — COMMENTED** [review]
And please update ChangeLog.md

**Comment by dolauli** [comment]
> And please update ChangeLog.md

No need to update changelog.md, since this is not a new version.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**